src/v/cluster/health_monitor_types.h

```diff
@@ -225,8 +225,21 @@ struct cluster_health_overview {
 
     // additional human readable information that will make debugging cluster
     // errors easier
+
+    // Zero or more "unhealthy" reasons, which are terse human-readable strings
+    // indicating one reason the cluster is unhealthy (there may be several).
+    // is_healthy is true iff this list is empty (effectively, is_healthy is
+    // redundnat but it's there for backwards compat and convenience).
```

Comment:
> (effectively, is_healthy is
 >   // redundnat but it's there for backwards compat and convenience)

Not sure I follow here, this is the internal type right so we don't really have any backwards compat reasons and could just provide 

```
bool is_healthy() const {
    return unhealthy_reasons.empty()
}
```

which we then call in admin_server.cc:4523?

Reply:
@StephanDollberg - I was more referring to the json API endpoint, i.e., we don't technically need the is_health member since a user can just do "is the unhealthy list length > 0". When writing this I forgot that this isn't the type directly used to generate the API, there's still one more level of indirection in admin_server as you point out, so yeah I could make this change.

Reply:
@StephanDollberg I updated the `cluster_health_overview` class to remove explicit healthy boolean and use `unhealthy_reasons.empty()` to implement `is_health`.